### PR TITLE
Docs - add how use custom disk

### PR DESCRIPTION
**Pull request by StephaneBour**
Hello,

I use a particular filesystem for images (https://github.com/sausin/laravel-ovh), and I needed to change the default disk.

I spent some time figuring out how to do this, so I completed the documentation for that.

Thank you.

### CODE REVIEW

File: docs/core/reference/images.md [hunk 1]
```diff
@@ -121,3 +121,19 @@ class YourCustomModel extends Model
 ```
 
 Now your models will auto generate transforms as defined in your configuration and still use medialibrary under the hood.
+
+## Use custom disk
+
+By default, media is stored on the `public` disk. If you want to use a different disk, you need to set the environment variable `MEDIA_DISK` to the name of the disk you want to use.
+
+```shell
+MEDIA_DISK=ovh
```
Comment (wychoong) [inline docs/core/reference/images.md]:
```suggestion
MEDIA_DISK=s3
```

use a more common disk as example

File: docs/core/reference/images.md [hunk 2]
```diff
@@ -121,3 +121,19 @@ class YourCustomModel extends Model
...
+```
+If you need more customisation, you can install `spatie/laravel-medialibrary` and publish the configuration file.
```
Comment (wychoong) [inline docs/core/reference/images.md]: the package already required by lunar, just need to publish the config file

Comment (StephaneBour) [reply docs/core/reference/images.md]: Hello, unfortunately it did not work when I launched the command. I had to include it in the project.

File: docs/core/reference/images.md (hunk 1 above)
Comment (StephaneBour) [reply docs/core/reference/images.md]: There is a tweak in the code to handle different s3s, so if you put it "s3" it works without configuring the MEDIA_DISK variable.
https://github.com/lunarphp/lunar/blob/68398b1920627e18051bc4d6a85bd52ab8408144/packages/admin/src/Http/Livewire/Traits/HasImages.php#L251

**Comment by StephaneBour** [comment]
I updated based on @wychoong 's comments, but I'm not supportive of these changes at now.

**Review by glennjacobs — APPROVED** [review]

**Comment by glennjacobs** [comment]
Thanks @StephaneBour

**Pull request merged**